client.go

```diff
@@ -196,6 +196,38 @@ func (c *Client) NewMessage(originator string, recipients []string, body string,
 	return message, nil
 }
 
+func (c *Client) NewMmsMessage(originator string, recipients []string, body string, mediaUrls []string, msgParams *MmsMessageParams) (*MmsMessage, error) {
```

Comment:
Since we only need either `body` or `mediaUrls`, it might be nicer to move them out of the function parameter list and into the `MmsMessageParams{}` struct.